user_docs/en/changes.t2t

```diff
@@ -70,8 +70,7 @@ There are many minor bug fixes for applications, such as Thunderbird, Adobe Read
   -
 - Component updates:
   - Updated LibLouis Braille translator to [3.29.0 https://github.com/liblouis/liblouis/releases/tag/v3.29.0]. (#16259, @codeofdusk)
-    - Added new detailed (with capital letters indicated) Belarusian and Ukrainian Braille tables, along with a Spanish table for reading Greek texts.
-    -
+    - Added new detailed (with capital letters indicated) Belarusian and Ukrainian Braille tables, a  Spanish table for reading Greek texts, and a  table for Lao Grade 1.

```

Comment:
This introduced some spacing issues and removed the list terminator. 
I've split this up into separate items for readibility

```suggestion
    - New detailed (with capital letters indicated) Belarusian and Ukrainian Braille tables.
    - New Spanish table for reading Greek texts.
    - New table for Lao Grade 1. (#16470)
    -
```